src/main/java/xolpoc/config/DeployerConfiguration.java

```diff
@@ -50,13 +67,30 @@ public ModuleDeployer moduleDeployer() {
 
 	@Bean
 	public ResourceModuleRegistry moduleRegistry() {
-		return new ResourceModuleRegistry(MODULE_HOME);
+		return new ResourceModuleRegistry(deployer.getModule());
 	}
 
 	@Bean
 	public ModuleFactory moduleFactory() {
 		return new ModuleFactory(moduleOptionsMetadataResolver());
 	}
+	
+	@Bean
+	@ConfigurationProperties("property")
+	public ModuleDeploymentProperties moduleDeploymentProperties() {
+		return new ModuleDeploymentProperties();
```

Comment:
Thanks @garyrussell. I added `@EnableIntegration` to the new `PluginConfiguration` class, and it's working fine now. @dsyer can correct me if I'm wrong, but I assume this worked before since the configuration of the MessageBus had been handled directly on `ModuleBootstrap` which is annotated with `@SpringBootApplication`.